packages/gasket-plugin-command/lib/ready.js

```diff
@@ -1,18 +1,15 @@
-/* eslint-disable no-unused-vars, no-sync */
-import { gasketBin, processCommand } from './cli.js';
-const isGasketCommand = /\/gasket\.(js|ts|cjs|mjs)$/;
+import { gasketBin } from './cli.js';
 
 /** @type {import('@gasket/core').HookHandler<'ready'>} */
-export default async function readyHook(gasket) {
-  const hasGasket = process.argv.some(arg => isGasketCommand.test(arg));
-
-  if (hasGasket) {
-    const cmds = await gasket.exec('commands');
-    cmds.forEach(cmd => {
-      const { command, hidden, isDefault } = processCommand(cmd);
-      gasketBin.addCommand(command, { hidden, isDefault });
-    });
-
+async function ready(gasket) {
+  gasket.isReady.then(() => {
     gasketBin.parse();
-  }
+  });
+}
+
+export default {
+  timing: {
+    first: true
+  },
+  handler: ready
 }
```

Comment:
I don't think it is necessary to set the timing of this hook
```suggestion
export default ready;
```